pulp_ostree/app/viewsets.py

```diff
@@ -119,7 +122,7 @@ class OstreeRefViewSet(ReadOnlyContentViewSet):
     filterset_class = RefContentFilter
 
 
-class CommitContentFilter(ContentFilter):
+class OstreeCommitContentFilter(ContentFilter):
```

Comment:
```suggestion
class OstreeCommitFilter(ContentFilter):
```
I think that would match the naming pattern (<classname>Filter) better.